controllers/git_tekton_resources_renovater_test.go

```diff
@@ -166,5 +152,22 @@ var _ = Describe("Git tekton resources renovater", func() {
 			Expect(allEvents[0].Reason).To(Equal("ErrorReadingPaCSecret"))
 			deleteComponent(componentNamespacedName)
 		})
+
+		It("It should not trigger job", func() {
+			installedRepositoryUrls := []string{
+				"https://github/test/repo1",
+				"https://github/test/repo2",
+			}
+			github.GetAppInstallations = func(appIdStr string, privateKeyPem []byte) ([]github.ApplicationInstallation, string, error) {
+				repositories := generateRepositories(installedRepositoryUrls)
+				return []github.ApplicationInstallation{generateInstallation(repositories)}, "slug", nil
+			}
+			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{componentKey: types.NamespacedName{Name: "testnottrigger"}, gitURL: "https://github/test/repo3"}))
+			createDefaultBuildPipelineRunSelector(defaultSelectorKey)
+			// try to wait for renovate Reconcile at least start
+			time.Sleep(2 * time.Second)
+			Eventually(listJobs).WithArguments(buildServiceNamespaceName).WithTimeout(timeout).Should(BeEmpty())
+			deleteComponent(componentNamespacedName)
```

Comment:
Better to move it into AfterEach

Reply:
I can't do that as not all tests are using it